tests/core/pyspec/eth2spec/test/helpers/rewards.py

```diff
@@ -159,7 +160,8 @@ def run_get_inactivity_penalty_deltas(spec, state):
             continue
 
         if finality_delay > spec.MIN_EPOCHS_TO_INACTIVITY_PENALTY:
```

Comment:
Now this function could be refactored with `spec.in_inactivity_leak(state)` helper.